packages/@adobe/spectrum-css-temp/components/table/skin.css

```diff
@@ -337,4 +341,105 @@ tbody.spectrum-Table-body {
     --spectrum-table-body-resize-indicator-background-color: Highlight;
     --spectrum-table-divider-border-color: CanvasText;
   }
+  
+  .spectrum-Table-headWrapper {
+    forced-color-adjust: none;
+    .spectrum-Table-headCell {
+      forced-color-adjust: auto;
+    }
+  }
+
+  .spectrum-Table-cell,
+  .spectrum-Table-headCell,
+  .spectrum-Table-headCellButton {
+    --spectrum-table-cell-border-color-key-focus: Highlight;
+    &:focus-ring,
+    &.is-focused {
+      &::before {
+        forced-color-adjust: none;
+      }
+    }
+  }
+
+  .spectrum-Table-columnResizer {
+    --spectrum-table-divider-border-color: CanvasText;
```

Comment:
there are so many selectors here, can we get all these variables moved up under just the top level one? more like checkbox's skin
it makes it way harder to make a mistake and forget one of these in unlikely selector combinations

Reply:
Unfortunately, some of the color overrides don't work without the selector specificity. I've moved all the forced-color mode color variable overrides to the top of the media query, separate from the `forced-color-adjust` stuff.